lib/kafka_ex/protocol/produce.ex

```diff
@@ -53,8 +53,14 @@ defmodule KafkaEx.Protocol.Produce do
     - value: is the message to be written to Kafka logs.
     - timestamp: timestamp (`kafka_version: "kayrock"` ONLY)
     """
-    defstruct key: nil, value: nil, timestamp: nil
-    @type t :: %Message{key: binary, value: binary, timestamp: integer}
+    defstruct key: nil, value: nil, headers: nil, timestamp: nil
+
+    @type t :: %Message{
+            key: binary,
+            value: binary,
+            headers: list(),
```

Comment:
should this be the same as in Fetch ?

```suggestion
            headers: [{key::binary, value::binary}],
```

Reply:
@jbruggem good catch it should be the same